src/awx_plugins/credentials/github_app.py

```diff
@@ -132,18 +132,19 @@ class EmptyKwargs(TypedDict):
 
 GH_CLIENT_ID_TRAILER_LENGTH = 16
 HEXADECIMAL_BASE = 16
+GH_CLIENT_ID_LENGTH_BOUNDS = (19, 22)
 
 
 def _is_intish(app_id_candidate: str | int) -> bool:
     return isinstance(app_id_candidate, int) or app_id_candidate.isdigit()
 
 
-def _is_client_id(client_id_candidate: str) -> bool:
-    client_id_prefix = 'Iv1.'
-    if not client_id_candidate.startswith(client_id_prefix):
+def _is_iv1_client_id(client_id_candidate: str) -> bool:
+    """Check if it's a valid Iv1 client ID (strict: Iv1. + 16 hex chars)."""
+    if not client_id_candidate.startswith('Iv1.'):
         return False
 
-    client_id_trailer = client_id_candidate.removeprefix(client_id_prefix)
+    client_id_trailer = client_id_candidate.removeprefix('Iv1.')
```

Comment:
No need to duplicate literals in multiple places. Revert this to the original forms.
```suggestion
    client_id_prefix = 'Iv1.'
    if not client_id_candidate.startswith(client_id_prefix):
        return False

    client_id_trailer = client_id_candidate.removeprefix(client_id_prefix)
```

Reply:
Marking this thread as resolved.